cmd/crioctl/main.go

```diff
@@ -0,0 +1,254 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net"
+	"os"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/containers/storage/pkg/archive"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+	"golang.org/x/net/context"
+
+	pb "github.com/cri-o/cri-o/pkg/private"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var checkpointCommand = cli.Command{
```

Comment:
can we merge this into crio-status or the other way around?

Reply:
I am open to whatever works best. The original idea behind crioctl came from @mrunalp to be able to test checkpoint/restore in CRI-O even before the CRI API changes are merged in Kubernetes. Now that the KEP has been merged it seems chances are high that the CRI API changes are merged and then we would not need the CRI-O only crioctl. We can just use crictl. I think the deadline for Kubernetes code changes for 1.24 is end of March so if checkpoint/restore is merged in Kubernetes we do not really need the whole crioctl setup. It was a workaround to be able to run tests.

Reply:
I integrated my changes into `crio-status` and renamed `crio-status` to `crioctl`. Please have a look. I hope CI still likes it.